lib/components/form/advanced-settings-panel.tsx

```diff
@@ -79,26 +73,57 @@ const HeaderContainer = styled.div`
 const Subheader = styled.h2`
   ${invisibleCss}
 `
-const baseColor = getBaseColor()
-const accentColor = baseColor || blue[900]
+
+const PlanTripButton = styled.button`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  gap: 0.5em;
+  background-color: var(--main-base-color, ${blue[900]});
+  border: 0;
+  width: 45%;
```

Comment:
super nit picky but from widths 992px - 1058px the trip text gets pushed to a new line. change the width to like 47% to fit the text on a single line.

Reply:
Really clean, thank you!!